docker/docker-compose.ci.yml

```diff
@@ -21,7 +21,7 @@ services:
     platform: linux/amd64
     build:
       context: ../
-      target: subgraph
+      target: dev
       args:
         BUILD_DIR: subgraph
 
```

Comment:
We actually have entries with custom command which assume they are inside the package folder such as `locksmith` and `locksmith-websub`. 

Ditto for some parts of CI scripts.

Reply:
yes the "production" step needs to be adapted as we dont use that yarn prod-install plugin anymore. The best approach is to move the workdir into the right folder and run the yarn command without workspace prefix then.

Reply:
144d0ae should fix it. wdyt?